.github/CODEOWNERS

```diff
@@ -70,6 +70,7 @@
 
 /scripts/ @thesonofthomp
 /tools/ @thesonofthomp
+/tools/test-harnesses @shaneeza
```

Comment:
Will this still work with line 72?

Reply:
I believe so. I think more specific paths take precedence